readme.md

```diff
@@ -1197,3 +1239,10 @@ $ calc
 
 Cool beans, 605 virtual machine instructions to compute pi to 1dp!
 
+
+<!---
+cfbolz: I think this is actually a nice place to end! I'd add a concluding sentence or two and be finished.
+
+adding a JIT for these expressions is not so obviously helpful, since we never
+execute a chunk twice, so producing machine code does not actually save time...
```

Comment:
Great I'll wrap it up here. Agree regarding the JIT - I did try in a different branch but figured the post was getting long enough and with so little to work with the JIT doesn't make a lot of sense!

Thanks for the review and tips.  I'll incorporate your comments and hopefully be done in the next few days.

Reply:
Awesome! Just let me know :-)

Reply:
All done! Moved to https://github.com/hardbyte/rpython-post so this repo can go back to trying to be a lox implementation :-)